docs/developing.rst

```diff
@@ -0,0 +1,185 @@
+
+.. code-block:: python
+
+    from .production import FEATURES
+     
+    FEATURES['ENABLE_SPECIAL_EXAMS'] = True
+     
+    PROCTORING_BACKENDS = {
+        'DEFAULT': 'mockprock',
+        'null': {},
+        'mockprock': {
+            'client_id': 'abcd',
+            'client_secret': 'abcdsecret',
+            'default_rules': {
+                'allow_snarfing': True,
+                'allow_grok': False
+            }
+        }
+    }
+ 
+    PROCTORING_SETTINGS = {
+        'MUST_BE_VERIFIED_TRACK': False
+    }
+
+Then back in your host shell::
+
+    cd ~/workspace/mockprock/
+    pip install -e .[server]
+    python -m mockprock.server
+
+The command will tell you you have to supply an client_id and client_secret. It'll open your browser to the django admin page where you should create or use an existing credential. Note the client_id and client_secret and restart the server::
+
+    python -m mockprock.server {client_id} {client_secret}
+
+How do I debug proctoring tests?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Set up a virtual environment, then::
+
+    cd /edx/src/edx-proctoring
+    make test-all
+
+You can now run the proctoring Python tests as follows::
+
+    cd /edx/src/edx-proctoring
+    # activate the proctoring virtualenv
+    source edx-proctoring/bin/activate
+    make test-python
+
+To debug these tests in PyCharm
+
+First ensure that you've set up PyCharm for debugging edx-platform
+See Debugging edX platform tests in PyCharm
+Add a new remote interpreter which uses the edx-proctoring virtual env:
+
+Follow the steps to create a remote interpreter Working with devstack and PyCharm
+
+Before saving the new interpreter, change the "Python interpreter path" to point to the virtual env: /edx/src/edx-proctoring/edx-proctoring/bin/python
+
+Note that this will have changed the default project interpreter to be your new one, so switch it back to your original
+Clone the "LMS Tests" and make the following changes
+Change "Script Parameters" to 
+
+test edx_proctoring
+Change the working directory to 
+
+/edx/src/edx-proctoring
+Add a path mapping: 
+
+Local path: /Users/YOUR_USERNAME/PATH/TO/edx-proctoring
+Remote path: /edx/src/edx-proctoring
+Change the Python interpreter to be your new edx-proctoring remote interpreter
+
+Verify the configuration
+
+Set a breakpoint in one of the proctoring tests
+Hit "Debug" to run the configuration
+You should see all the tests running, and then at some point the breakpoint should be hit
+
+How do I set up proctoring on a sandbox?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Start by following the steps here: https://github.com/edx/edx-proctoring
+
+* Add the edX-specific configuration settings
+* Restart Studio and LMS::
+
+    sudo /edx/bin/supervisorctl restart lms cms
+
+* Create a test course
+
+* Enroll verified@example.com in the course
+* Log in to Django admin
+* Add a verified course mode for your course
+* Update the verified user's mode to be "verified"
```

Comment:
This is actually easier said than done (the django admin section for course enrollments, where the mode this is referring to is, is hidden by a waffle flag). I'd probably recommend a new developer to use an admin user with masquerading instead, especially if they're just getting started.